Podfile.lock

```diff
@@ -152,6 +152,6 @@ SPEC CHECKSUMS:
   XCGLogger: 0434f15e3909cdc450bb63faf638b8792ab782ab
   yoga: 684513b14b03201579ba3cee20218c9d1298b0cc
 
-PODFILE CHECKSUM: 1f766265a166d90264d5898db0d25c1c99edf4d9
+PODFILE CHECKSUM: 424770196b2549c8a3957ac3cbd6af5e1de5cd9d
```

Comment:
This should not happen. Are you sure you run right version of Cocoapods?

Reply:
i use bundle exec pod install, I assume it uses the correct one

Reply:
I will check why it is changed